registers/README.md

```diff
@@ -0,0 +1,14 @@
+# A Trillian tutorial in seven diffs
+
+This tutorial shows how to build a demo application using Trillian that creates a log, and from the log,
+creates a map, and then serves that map from a webserver.
+
+The master branch contains the completed project. The seven steps can be viewed as diffs here:
+
+1. [Getting the data](https://github.com/benlaurie/trillian-examples/compare/register-0...benlaurie:register-1?expand=1).
```

Comment:
Update github URLs to compare upstream branches (ready for when this lands?)

Reply:
When I have more brain.